core/common/src/main/java/alluxio/worker/block/BlockStoreMeta.java

```diff
@@ -17,50 +17,19 @@
 import java.util.List;
 import java.util.Map;
 
-import javax.annotation.concurrent.ThreadSafe;
-
 /**
  * Interface for the block store meta in Alluxio.
  */
 public interface BlockStoreMeta {
-
-  /**
-   * Factory for {@link BlockStoreMeta}.
-   */
-  @ThreadSafe
-  class Factory{
-
-    private Factory() {}
-
-    /**
-     * Factory method to return a BlockStoreMeta instance without blockIds.
-     *
-     * @param manager the BlockMetadataManager
-     * @return BlockStoreMeta instance
-     */
-    public static BlockStoreMeta create(BlockMetadataManager manager) {
-      return new DefaultBlockStoreMeta(manager, false);
-    }
-
-    /**
-     * Factory method to return a BlockStoreMeta instance with blockIds.
-     *
-     * @param manager the BlockMetadata Manager
-     * @return BlockStoreMeta instance
-     */
-    public static BlockStoreMeta createFull(BlockMetadataManager manager) {
-      return new DefaultBlockStoreMeta(manager, true);
-    }
-  }
   /**
-   * Note: This is only available in {@link BlockStoreMeta.Factory#createFull}.
+   * Note: This is only available when blocks are included.
```

Comment:
Not very clear what when blocks are included mean?

Reply:
@LuQQiu  agree. I may submit another PR to improve it. But I didn't do it here to avoid further increase the footprint of this PR